sdk/monitor/monitor-opentelemetry/src/metrics/quickpulse/utils.ts

```diff
@@ -232,8 +233,8 @@ function getRequestData(span: ReadableSpan): RequestData {
       try {
         const urlObj = new URL(requestData.Url);
         requestData.Name = `${httpMethod} ${urlObj.pathname}`;
-      } catch (ex) {
-        /* no-op */
+      } catch (ex: any) {
+        Logger.getInstance().info("Request data sent to live metrics has no valid URL field.", ex);
```

Comment:
We know that this is going to happen 100% of the time when the customer is using NextJS, do we have a mechanism to throttle these information messages, as I can see someone complaining (including the ICM customer) that this is awfully noisy)